app/services/manage_courses_api.rb

```diff
@@ -0,0 +1,36 @@
+module ManageCoursesAPI
+  class Connection
+    BASE_URL = Settings.manage_api.base_url
+    TOKEN = Settings.manage_api.secret
+
+    def self.api
+      Faraday.new(url: BASE_URL) do |faraday|
+        faraday.response :logger unless Rails.env.test?
+        faraday.adapter Faraday.default_adapter
+        faraday.headers['Content-Type'] = 'application/json'
+        faraday.headers['Authorization'] = "Bearer #{TOKEN}"
+      end
+    end
+  end
+
+  class Request
+    class << self
```

Comment:
Is this inheritance?

Reply:
This is making us not have to specify `def self.method` for every method underneath; basically saying "everything in this block is `static`"